Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Created by .ignore support plugin (hsz.mobi)\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .gitignore	(revision 9862c9614f9e5c342d5465716fc3a4d19eac0c95)
+++ .gitignore	(date 1598393109257)
@@ -1,1 +1,1 @@
-# Created by .ignore support plugin (hsz.mobi)
+*.idea
\ No newline at end of file
